python/gui/auto_generated/qgisinterface.sip.in

```diff
@@ -632,6 +632,15 @@ Take screenshots for user documentation
 .. versionadded:: 3.4
 %End
 
+    virtual QgsLayerTreeRegistryBridge::InsertionPoint layerTreeInsertionPoint() = 0;
+%Docstring
+Returns the insertion group with its insertion index
```

Comment:
Can you expand on this, copying part of the setLayerInsertionPoint dox? E.,g. 

   This represents the current layer tree group and index where newly added map layers should be inserted into.